src/ophys_etl/workflows/on_prem/dags/ophys_processing.py

```diff
@@ -51,7 +50,7 @@
     dag_id="ophys_processing",
     schedule=None,
     catchup=False,
-    start_date=datetime.datetime.now(),
+    start_date=pendulum.yesterday(),
```

Comment:
What's the logical consequence of the static time per trigger? One thing I'm trying to work through is how this would effect if a job fails and we need to reprocess. I assume the run_id would save us from any collisions in the airflow db and we would have to manually trigger at that point as well. So it would be pretty safe I assume?

Reply:
I'm not exactly following your concern. This is the start time of the dag. As far as I can tell it really serves no purpose other than it causes problems like I mentioned above. If a dag run is before this start time then it won't run which does not seem possible if the dag is triggered after this start time.

Reply:
Ah, okay. Thought it was used additionally in our triggers but guess not. Looks all good then.